samples-generator/lib/saveJson.js

```diff
@@ -1,25 +1,32 @@
 'use strict';
 var Cesium = require('cesium');
 var fsExtra = require('fs-extra');
+var saveBinary = require('./saveBinary');
 
 var defaultValue = Cesium.defaultValue;
 
-module.exports = saveTilesetJson;
+module.exports = saveJson;
 
 /**
- * Save a tileset.json file to disk.
+ * Save a json file to disk.
  *
  * @param {String} path The path to save the tileset.json.
  * @param {Object} json The JSON.
  * @param {Boolean} [prettyJson=true] Whether to prettify the JSON.
+ * @param {Boolean} [gzip=false] Whether to use GZIP or not
  *
  * @returns {Promise} A promise that resolves when the tileset.json is saved.
  */
-function saveTilesetJson(path, json, prettyJson) {
+function saveJson(path, json, prettyJson, gzip) {
     prettyJson = defaultValue(prettyJson, true);
     var options = {};
     if (prettyJson) {
         options.spaces = 2;
     }
+    gzip = defaultValue(gzip, false);
+    if (gzip) {
+        return saveBinary(path, Buffer.from(json), gzip);
```

Comment:
```suggestion
        return saveBinary(path, Buffer.from(JSON.stringify(json)), gzip);
```

If gzip is true it crashes without this fix.